backend_tv/lifter.cpp

```diff
@@ -1071,7 +1081,7 @@ class arm2llvm {
       globalVar = glob->second;
     }
 
-    return globalVar;
+    return make_pair(globalVar, storePtr);
```

Comment:
I think it's bad style to mix functions for pairs and tuples, just use pairs or tuples everywhere.